### ISSUE TITLE
HTTP method matching

### ISSUE THREAD

**Issue by arietto** [cutelyst/cutelyst#151]
Is there any alternative to :ActionClass(REST) in Cutelyst for HTTP method matching?
https://stackoverflow.com/a/20914232
http://www.catalystframework.org/calendar/2013/3
Sometimes I have to make a POST request with query embedded in body for getting things. I mean that this method is a getter (logically), so I'd like to have an alias.
POST get_stuff
instead of stuff_POST.
So, I don't want Controller methods to contain HTTP postfix (like "_POST"), I'd like to have an attribute instead.

**Comment by dantti** [comment]
No, this feature is not available in Cutelyst at the moment. You can do a PR so that I can review or wait for to have a look at how to implement this without sacrificing performance of current requests.
